src/stylesheets/core/mixins/_external-link.scss

```diff
@@ -4,15 +4,16 @@
   $image-path: $theme-image-path
 ) {
   &::after {
-    $icon-size: 0.65em;
+    $external-link-size: 1.75ex;
     background-image: url("#{$image-path}/#{$external-link}.svg");
-    background-position: 50% 60%;
     background-repeat: no-repeat;
     background-size: 100%;
     content: "";
-    display: inline;
+    display: inline-block;
+    height: $external-link-size;
     margin-left: units(2px);
-    padding-left: $icon-size;
```

Comment:
Removing `inline` and `padding-left` reverts the changes we did to fix the icon breaking into next line [#3395]. 

Testing:
```html
<p>is an alternate external text link on a dark background. is an alternate external text link on a dark background.     is an alternate external text link on a. is an alternate external text link on a dark background.     is an alternate <a class="usa-link usa-link--alt usa-link--external" href="https://i.giphy.com/media/WPzQF6ruiIIVzHNlwX/source.gif">alternate external text link on a dark background</a></p>
```

Produces:
![image](https://user-images.githubusercontent.com/3385219/127665276-abb8730f-0673-46a4-a288-0af9bd3b5d60.png)

Reply:
Ah, thanks — I'll see if I can resolve this

Reply:
I'll leave a comment in the code to explain the reasoning for that technique

Reply:
Sounds good! Resolved